src/Views/Shared/_Layout.cshtml

```diff
@@ -34,11 +34,19 @@
   <script src="~/application.js" asp-append-version="true"></script>
 }
 
+@section footerTop {
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <p class="govuk-footer__content">For advice about teaching or teacher training you can call <a href="https://getintoteaching.education.gov.uk/" class="govuk-footer__link">Get into Teaching</a> on Freephone 0800 389 2500 or chat to an adviser using their <a href="https://ta-chat.education.gov.uk/chat/chatstart.aspx?domain=www.education.gov.uk&department=GetIntoTeaching%27,%27new_win%27,%27width=0,height=0%27);return%20false;%22&SID=1" class="govuk-footer__link">online chat service</a> between 8am and 8pm, Monday to Friday.</p>
```

Comment:
We can do a `<a href="tel:08003892500">0800 389 2500</a>` for the phone link.

minor

Reply:
I do this on my personal projects but GDS advise against it, see https://design-system.service.gov.uk/patterns/telephone-numbers/#don-t-display-telephone-numbers-as-links-on-devices-that-can-t-make-calls

Reply:
I actually thought about that and was wondering if there's guidance. Good!